feat/DPP-89 changed configuration for dbc

sksadjad opened the pull request.

Commit:
DPP-84 Updated the configuration files

Commit:
DPP-84 Removed doubled sequence property from `/home/maikel/IdeaProjects/OID4VC-demo/packages/oid4vci-demo-frontend/src/configs/triall.json`

sanderPostma committed:
Merge remote-tracking branch 'origin/develop' into feature/DPP-84

# Conflicts:
#	packages/oid4vci-demo-frontend/src/configs/triall.json

sanderPostma committed:
DPP-84: moved demos back into repo in a more ordered manner

sanderPostma committed:
DPP-84: work on restoring DBC

sanderPostma committed:
Merge remote-tracking branch 'origin/develop' into feature/DPP-84

# Conflicts:
#	packages/oid4vci-demo-frontend/src/pages/SSIInformationRequestPage/index.tsx

sanderPostma committed:
DPP-84: added sphereonWalletIdentity.json

sanderPostma committed:
Merge remote-tracking branch 'origin/develop' into feature/DPP-84

sanderPostma committed:
Added missing banqup configs

sanderPostma committed:
Merge remote-tracking branch 'origin/develop' into feature/DPP-84

sanderPostma committed:
Fixed sphereonGuest.json

sanderPostma committed:
Fixed more jsons

sanderPostma committed:
Fixed Voornaam -> firstName etc

sanderPostma committed:
Merge branch 'feature/VDX-252' into feature/DPP-84

# Conflicts:
#	packages/agent/conf/demos/pf3/templates/sphereonGuest/GuestCredential.hbs
#	packages/agent/conf/demos/sphereonGuest/oid4vci_metadata/sphereonGuest.json
#	packages/oid4vci-demo-frontend/src/configs/triall.json
#	packages/oid4vci-demo-frontend/src/pages/SSIInformationRequestPage/index.tsx
#	packages/oid4vci-demo-frontend/src/pages/SSIInformationSuccessPage/index.tsx
#	packages/oid4vci-demo-frontend/src/pages/SSISelectCredentialPage/index.tsx

sanderPostma committed:
DPP-84: Triall DIDs

sanderPostma committed:
Merge remote-tracking branch 'origin/develop' into feature/DPP-84

# Conflicts:
#	packages/oid4vci-demo-frontend/src/configs/dbc.json
#	packages/oid4vci-demo-frontend/src/configs/triall.json

Commit:
VDX-278 Fixed Notary web demo issues

sanderPostma committed:
DPP-84: Fixed /information/success page

sksadjad committed:
feat: changed configuration for dbc

sksadjad committed:
feat: changed shareSuccess image for dbc

sksadjad committed:
feat: added image

sksadjad committed:
feat: removed hardcoded dbc credential dataSupplier

Commit:
VDX-278 Verify Credential Page, added the missing Text component

sksadjad committed:
fix: fixed navigation after issue for dbc

sksadjad committed:
fix: changed the dbc credential template

sksadjad committed:
fix: changed the dbc presentation_definition to SphereonWalletIdentityCredential

Commit:
VDX-278 Credential Verify Request Page, fixed the text

sanderPostma commented on packages/agent/conf/demos/dbc/presentation_definitions/SphereonWalletIdentityCredential.json:
```diff
@@ -0,0 +1,16 @@
+{
+  "id": "dbc2023",
```
Since we are asking for a SphereonWalletIdentityCredential, shouldn't it be "sphereonWalletIdentity"?
If so, then also rename dbcConferenceAttendee.json to SphereonWalletIdentityCredential.json in oid4vp_options and change dbc2023 to sphereonWalletIdentity
It's dbcConferenceAttendee we are issueing

sksadjad replied:
changed to `sphereonWalletIdentity` and also changed related places

sanderPostma reviewed (changes requested):
I've seen the config working, but just check the oid4vp_options

sksadjad committed:
fix: changed oid4vp ids for dbc to sphereonWalletIdentity

sanderPostma reviewed (approved):
Don't merge yet, going to rebase

sanderPostma committed:
DPP-89: Fixed start:prod startup

sanderPostma committed:
Merge remote-tracking branch 'origin/feature/DPP-89' into feature/DPP-89

sanderPostma committed:
DPP-89: Fixing start:prod startup

sanderPostma committed:
DPP-89: Fixing start:prod startup

BtencateSphereon commented on packages/agent/conf/demos/dbc/oid4vci_metadata/dbc.json:
```diff
@@ -59,39 +59,39 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/dbc/oid4vci_metadata/dbc.json:
```diff
@@ -59,39 +59,39 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/dbc/presentation_definitions/SphereonWalletIdentityCredential.json:
```diff
@@ -0,0 +1,16 @@
+{
```
we already have a sphereonWalletIdentity.json, so what is the difference between this one and the one we already have?

BtencateSphereon replied:
hmmm, what is the logic behind the conf folder for the agent? this places a dbc conf folder, which i think is fine. But then we should apply this logic to all demo's. And then the question becomes. how do we handle duplicates? i am not sure of managing duplicates is a nice approach.

sanderPostma replied:
This is applied for all demos, see the moved files.
For some files like the dids we must have different files per ecosystem. Like with a deployment for knb/kvk we can't have the files from other ecosystems.
Also an agent running for banqup has no business running configs for knb etc.
Maintaining duplicates is not ideal in all cases, however in other cases you might need a differences in files between the ecosystems like a different PD text for the same credential. Isolating the configs per ecosystem also makes sure everything is testable for that specific ecosystem. It has everything it needs to run a deployment nothing more, nothing less.

BtencateSphereon commented on packages/agent/conf/demos/dbc/templates/dbc/DBCConferenceAttendee.hbs:
```diff
@@ -0,0 +1,19 @@
+"email": "{{email}}",
+"event": {
+"name": "DBC Conference 2023",
+"date": "2023-06-26"
```
Should this date not de provided?

sanderPostma replied:
Good question, but where should we configure this else? the .hbs file is already a conf file. 
Hidden form  fields would work but is not a nice. Extra parameters in the create-credential-offer route step could be, but is that better than in conf/templates?
Maybe create a separate issue for this as this is beyond the scope of DPP-89.

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/dbc/templates/dbc/DBCConferenceAttendee.hbs:
```diff
@@ -0,0 +1,19 @@
+"lastName": "{{lastName}}",
+"email": "{{email}}",
+"event": {
+"name": "DBC Conference 2023",
```
Should this name not de provided? also niels mentioned we should not add years. just checking if this is what we want

sanderPostma replied:
Good question, but where should we configure this else? the .hbs file is already a conf file.
Hidden form fields would work but is not a nice. Extra parameters in the create-credential-offer route step could be, but is that better than in conf/templates?
Maybe create a separate issue for this as this is beyond the scope of DPP-89.

BtencateSphereon commented on packages/agent/conf/demos/energy-shr/oid4vci_metadata/energyshr.json:
```diff
@@ -59,40 +59,40 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/energy-shr/oid4vci_metadata/energyshr.json:
```diff
@@ -59,40 +59,40 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/dbc/oid4vci_metadata/dbc.json:
```diff
@@ -59,39 +59,39 @@
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```
this is the default branding when there is no locale. it is a dutch conference. should we not set this to dutch?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/dbc/oid4vci_metadata/dbc.json:
```diff
@@ -59,39 +59,39 @@
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```
this is the default branding when there is no locale. it is a dutch conference. should we not set this to dutch?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/energy-shr/oid4vci_metadata/energyshr.json:
```diff
@@ -59,40 +59,40 @@
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```
what should be the default locale here? dutch or english?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/energy-shr/oid4vci_metadata/energyshr.json:
```diff
@@ -59,40 +59,40 @@
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```
what should be the default locale here? dutch or english?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

sanderPostma committed:
DPP-89: Reverted Fixing start:prod

BtencateSphereon commented on packages/agent/conf/demos/fmdm/oid4vci_metadata/fmdm.json:
```diff
@@ -59,40 +59,40 @@
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```
what should be the default locale here? dutch or english?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/fmdm/oid4vci_metadata/fmdm.json:
```diff
@@ -59,40 +59,40 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/fmdm/oid4vci_metadata/fmdm.json:
```diff
@@ -59,40 +59,40 @@
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```
what should be the default locale here? dutch or english?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon replied:
we support english and dutch for now, but i believe some parties want dutch as default and others english. with these comments i am just making sure we get that correct

BtencateSphereon commented on packages/agent/conf/demos/fmdm/oid4vci_metadata/fmdm.json:
```diff
@@ -59,40 +59,40 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/auckland/oid4vci_metadata/knb_van_auckland.json:
```diff
@@ -65,18 +65,18 @@
             "display": [
               {
-                "name": "Omschrijving"
+                "name": "Description"
```
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

sanderPostma replied:
-> question overwritten

BtencateSphereon commented on packages/agent/conf/demos/knb/auckland/oid4vci_metadata/knb_van_auckland.json:
```diff
@@ -65,18 +65,18 @@
               },
               {
-                "name": "Omschrijving",
+                "name": "Beschrijving",
```
what is the reason for this change?

sanderPostma replied:
I did that, but nfi. reverted...

BtencateSphereon commented on packages/agent/conf/demos/knb/auckland/oid4vci_metadata/knb_van_auckland.json:
```diff
@@ -130,7 +130,7 @@
           }
         },
-        "id": "knb-notary",
+        "id": "VanAuckland2023",
```
i think niels mentioned that we should not add any years. just checking that here

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/auckland/templates/auckland/NotaryCompanyVerification.hbs:
```diff
@@ -8,7 +8,7 @@
     "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
     "credentialSubject": {
-        "Omschrijving": "Bedrijf is geverifieerd door de notaris",
+        "description": "Company has been verified by the notary",
```
are we sure this should now be english?

sanderPostma replied:
I reverted this along with Voornaam -> firstName
Actually this would be a correct i18n way:
```
{
    "@context": [
        "https://www.w3.org/2018/credentials/v1", {
            "description": {
                "@id": "http://schema.org/description",
                "@container": "@language"
            },
        }
    ],
    "type": [
        "VerifiableCredential",
        "NotaryCompanyVerification"
    ],
    "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
    "credentialSubject": {
        "description": {
          "en": "Company has been verified by the notary",
          "nl": "Bedrijf is geverifieerd door de notaris"
         }
        {{{mergeJSON @root}}}
    }
}
```

Does our wallet support that? Do we just dump the credentialSubject 1:1 or do we actually read from it?

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/oid4vci_metadata/knb_babylon.json:
```diff
@@ -1,9 +1,9 @@
 {
-  "correlationId": "https://bank.demo.sphereon.com/agent",
+  "correlationId": "http://192.168.1.200:5000",
```
local ip

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/oid4vci_metadata/knb_babylon.json:
```diff
@@ -1,9 +1,9 @@
   "metadata": {
-    "credential_issuer": "https://bank.demo.sphereon.com/agent",
-    "credential_endpoint": "https://bank.demo.sphereon.com/agent/credentials",
+    "credential_issuer": "http://192.168.1.200:5000",
```
local ip

sanderPostma replied:
wt... fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/oid4vci_metadata/knb_babylon.json:
```diff
@@ -1,9 +1,9 @@
-    "credential_issuer": "https://bank.demo.sphereon.com/agent",
-    "credential_endpoint": "https://bank.demo.sphereon.com/agent/credentials",
+    "credential_issuer": "http://192.168.1.200:5000",
+    "credential_endpoint": "http://192.168.1.200:5000/credentials",
```
local ip

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/oid4vci_metadata/knb_babylon.json:
```diff
@@ -130,7 +130,7 @@
           }
         },
-        "id": "knb-bank",
+        "id": "BabylonBank2023",
```
i think niels mentioned that we should not add any years. just checking that here

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/oid4vci_options/knb_van_auckland.json:
```diff
@@ -1,6 +1,6 @@
-  "definitionId": "knb-bank",
-  "correlationId": "https://bank.demo.sphereon.com/agent",
+  "definitionId": "knb-babylon",
+  "correlationId": "http://192.168.1.200:5000",
```
local ip

nklomp replied:
Not only that. The whole name change is worse. Just stick to the roles for the names, not the imaginary names we created. So knb-bank is way better than knb-bavylon, which says nothing to most people

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/oid4vp_options/knb_babylon.json:
```diff
@@ -1,6 +1,6 @@
 {
   "definitionId": "knb-babylon",
-  "correlationId": "https://bank.demo.sphereon.com/agent",
+  "correlationId": "http://192.168.1.200:5000",
```
local ip

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/presentation_definitions/babylon.json:
```diff
@@ -0,0 +1,16 @@
+{
+  "id": "knb-babylon",
+  "purpose": "We hebben informatie nodig van u",
```
checking here that dutch is the locale we want

sanderPostma replied:
Maarten requested interactions in Dutch  yes

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/presentation_definitions/babylon.json:
```diff
@@ -0,0 +1,16 @@
+    {
+      "id": "01aba20d-6b8d-4ad0-95ae-28ec4267179f",
+      "name": "Rekening openen bij Babylon",
+      "purpose": "We hebben informatie verstrekt door het KvK nodig",
```
this does not read very nicely. something like:

We hebben inforrmatie nodig dat is verstrekt door de KvK?

sanderPostma replied:
ok
We hebben de door de KvK verstrekte informatie nodig.

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/templates/babylon/BankAccount.hbs:
```diff
@@ -7,8 +7,7 @@
     "credentialSubject": {
-        "Omschrijving": "Bankrekening bij Babylon",
-        "Rekeningnummer": "NL31 BABY 0123456789",
+        "description": "Company has a bank account at Babylon",
```
we sure this should be english?

sanderPostma replied:
see prev answer Auckland

BtencateSphereon commented on packages/agent/conf/demos/knb/babylon/templates/babylon/BankAccount.hbs:
```diff
@@ -7,8 +7,7 @@
     ],
     "credentialSubject": {
-        "Omschrijving": "Bankrekening bij Babylon",
-        "Rekeningnummer": "NL31 BABY 0123456789",
```
why was this removed?

sanderPostma replied:
Omschrijving is reverted back to description
Rekeningnummer is auto-generated in the form now

BtencateSphereon commented on packages/agent/conf/demos/knb/kvk/oid4vci_metadata/knb_kvk.json:
```diff
@@ -49,11 +49,11 @@
             },
             "background_image": {
-              "url": "https://ibb.co/L6DR6L8",
+              "url": "https://i.ibb.co/K7ch7MX/kvk-bg.png",
```
this image has rounded corners. it should not have that. we should fix it, because i remember it looking bad in the demo.

check all images! cause i randomly picked this one to test and got lucky

sanderPostma replied:
Placed screenshot of this in DPP-84. Actually DPP-89 was only meant for DBC, but all these have to be fixed this sprint anyways.

BtencateSphereon commented on packages/agent/conf/demos/knb/kvk/presentation_definitions/kvk.json:
```diff
@@ -0,0 +1,16 @@
+    {
+      "id": "4cf7cff1-0234-4f35-9d21-251668a60950",
+      "name": "Verificatie notaris",
+      "purpose": "We hebben informatie verstrekt door uw notaris nodig",
```
this does not read very nicely. something like:

We hebben informatie nodig dat is verstrekt door uw notaris?

sanderPostma replied:
We hebben de door de KvK verstrekte informatie nodig.

BtencateSphereon commented on packages/agent/conf/demos/knb/kvk/templates/kvk/KvkInschrijving.hbs:
```diff
@@ -6,9 +6,9 @@
+    "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
     "credentialSubject": {
-        "Omschrijving": "KvK inschrijving",
+        "description": "Company registered with the KvK",
```
is this the correct language?

sanderPostma replied:
Maarten request Dutch for this demo

sanderPostma replied:
I reverted this along with Voornaam -> firstName
Actually this would be a correct i18n way:

{
    "@context": [
        "[https://www.w3.org/2018/credentials/v1",](https://www.w3.org/2018/credentials/v1%22,) {
            "description": {
                "@id": "[http://schema.org/description",](http://schema.org/description%22,)
                "@container": "@language"
            },
        }
    ],
    "type": [
        "VerifiableCredential",
        "NotaryCompanyVerification"
    ],
    "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
    "credentialSubject": {
        "description": {
          "en": "Company has been verified by the notary",
          "nl": "Bedrijf is geverifieerd door de notaris"
         }
        {{{mergeJSON @root}}}
    }
}

Does our wallet support that? Do we just dump the credentialSubject 1:1 or do we actually read from it?

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -140,40 +140,40 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -140,40 +140,40 @@
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```
what should be the default locale?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -140,40 +140,40 @@
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```
what should be the default locale?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -140,40 +140,40 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -277,39 +277,39 @@
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```
what should be the default locale?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -277,39 +277,39 @@
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```
what should be the default locale?

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -277,39 +277,39 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

BtencateSphereon commented on packages/agent/conf/demos/pf3/oid4vci_metadata/pf3.json:
```diff
@@ -277,39 +277,39 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/knb/kvk/presentation_definitions/kvk.json:
```diff
@@ -0,0 +1,16 @@
+{
+  "id": "knb-kvk",
+  "purpose": "We hebben informatie nodig van u",
```
is dutch the language we want?

sanderPostma replied:
yes, see prev answer

sanderPostma replied:
Maarten request Dutch for this demo

BtencateSphereon commented on packages/agent/conf/demos/pf3/presentation_definitions/energyshrGuest.json:
```diff
@@ -0,0 +1,30 @@
+{
+  "id": "energySHR2023",
```
i think niels mentioned that we should not add any years. just checking that here

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/pf3/presentation_definitions/energyshrGuest.json:
```diff
@@ -0,0 +1,30 @@
+{
+  "id": "energySHR2023",
+  "purpose": "For this portal we need your e-mail address and name from an EnergySHR guest or DBC Conference credential",
```
check locale

sanderPostma replied:
placed screenshot in DPP-84

BtencateSphereon commented on packages/agent/conf/demos/pf3/presentation_definitions/fmaGuest.json:
```diff
@@ -0,0 +1,30 @@
+{
+  "id": "fma2023",
```
i think niels mentioned that we should not add any years. just checking that here

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/pf3/presentation_definitions/fmaGuest.json:
```diff
@@ -0,0 +1,30 @@
+{
+  "id": "fma2023",
+  "purpose": "For this portal we need your e-mail address and name from a FMA guest or DBC Conference credential",
```
check locale

sanderPostma replied:
Placed screenshot in DPP-84

BtencateSphereon commented on packages/agent/conf/demos/sphereonGuest/oid4vci_metadata/sphereonGuest.json:
```diff
@@ -143,10 +143,10 @@
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```
check what default locale should be

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/sphereonGuest/oid4vci_metadata/sphereonGuest.json:
```diff
@@ -159,10 +159,10 @@
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```
check what default locale should be

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/sphereonGuest/presentation_definitions/sphereonGuest.json:
```diff
@@ -0,0 +1,30 @@
+{
+  "id": "sphereonGuest",
+  "purpose": "For this portal we need your e-mail address and name from a Sphereon guest credential",
```
check the locale

sanderPostma replied:
Placed screenshot in DPP-84

BtencateSphereon commented on packages/agent/conf/demos/triall/oid4vci_metadata/triall.json:
```diff
@@ -56,40 +56,40 @@
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```
check locale

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/triall/oid4vci_metadata/triall.json:
```diff
@@ -56,40 +56,40 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
was fixed

BtencateSphereon commented on packages/agent/conf/demos/triall/oid4vci_metadata/triall.json:
```diff
@@ -56,40 +56,40 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/triall/oid4vci_metadata/triall.json:
```diff
@@ -56,40 +56,40 @@
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```
check locale

sanderPostma replied:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -57,7 +57,7 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -73,7 +73,7 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -158,7 +158,7 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -174,7 +174,7 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -259,8 +259,8 @@
           "companyName",
-          "contactFirstName",
-          "contactLastName",
+          "contactFirst Name",
```
no spaces, it's a property name

sanderPostma replied:
Fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -259,8 +259,8 @@
-          "contactFirstName",
-          "contactLastName",
+          "contactFirst Name",
+          "contactLast Name",
```
no spaces, it's a property name

sanderPostma replied:
Fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -281,7 +281,7 @@
             ]
           },
-          "contactFirstName": {
+          "contactFirst Name": {
```
no spaces, it's a property name

sanderPostma replied:
Fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/oid4vci_metadata/banqup.json:
```diff
@@ -297,7 +297,7 @@
             ]
           },
-          "contactLastName": {
+          "contactLast Name": {
```
no spaces, it's a property name

sanderPostma replied:
Fixed

BtencateSphereon commented on packages/agent/conf/demos/unified-post/banqup/templates/banqup/LegalEntity.hbs:
```diff
@@ -8,8 +8,8 @@
-        "contactFirstName": "{{firstName}}",
-        "contactLastName": "{{lastName}}",
+        "contactfirstName": "{{firstName}}",
+        "contactLast Name": "{{lastName}}",
```
no spaces, it's a property name

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/docker/oid4vci_metadata/sphereon.json:
```diff
@@ -148,14 +148,14 @@
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
```
Voornaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/agent/conf/docker/oid4vci_metadata/sphereon.json:
```diff
@@ -164,14 +164,14 @@
               },
               {
-                "name": "Achternaam",
+                "name": "lastName",
```
Achternaam

sanderPostma replied:
fixed

BtencateSphereon commented on packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json:
```diff
@@ -64,8 +64,8 @@
   "ssi_information_request_page_form_company_location_label": "Locatie van bedrijf",
-  "ssi_information_request_page_form_first_name_label": "Voornaam",
-  "ssi_information_request_page_form_last_name_label": "Achternaam",
+  "ssi_information_request_page_form_first_name_label": "firstName",
```
this is a dutch locale file

sanderPostma replied:
fixed

BtencateSphereon commented on packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json:
```diff
@@ -64,8 +64,8 @@
-  "ssi_information_request_page_form_first_name_label": "Voornaam",
-  "ssi_information_request_page_form_last_name_label": "Achternaam",
+  "ssi_information_request_page_form_first_name_label": "firstName",
+  "ssi_information_request_page_form_last_name_label": "lastName",
```
this is a dutch locale file

sanderPostma replied:
fixed

BtencateSphereon commented on packages/oid4vci-demo-frontend/src/pages/SSIInformationManualRequestPage/index.tsx:
```diff
@@ -13,13 +13,23 @@ import {useFlowRouter} from "../../router/flow-router"
+  if (!formConfig.form) {
+
+      formConfig.form = DEFAULT_FORM
+      return {
```
if no form is in the config, it was discussed to not show anything. you just need to supply a form if you want 1

sanderPostma replied:
fixed

BtencateSphereon commented on packages/oid4vci-demo-frontend/src/types/default-form.ts:
```diff
@@ -0,0 +1,32 @@
+import {DataFormRow} from "../ecosystem/ecosystem-config"
```
there is no default form

sanderPostma replied:
deleted

BtencateSphereon reviewed (changes requested):
well....replace all can be nice, but it surely was not here. also i am not sure if duplicating stuff is optimal here. for example we duplicate the sphereonWalletIdentity pd. and we also need to maintain that

sanderPostma committed:
DPP-89: PR feedback

sanderPostma committed:
DPP-89: ssi.dutchblockchaincoalition.org/issuer -> ssi.dutchblockchaincoalition.org/agent

sanderPostma committed:
DPP-89: reverted Beschrijving

nklomp commented:
Also revert changes for instance that are knb-bank -> babylon in identifiers and configs please. These change make 0 sense, as babylon is just a made up name that says nothing to most people. The configs had roles in their names, stick with that please

sanderPostma committed:
DPP-89: PR feedback

sanderPostma committed:
DPP-89: PR feedback

sanderPostma committed:
DPP-89: PR feedback

sanderPostma committed:
DPP-89: PR feedback

sanderPostma committed:
DPP-89: PR feedback

sanderPostma commented:
> Also revert changes for instance that are knb-bank -> babylon in identifiers and configs please. These type of changes make 0 sense, as babylon is just a made up name that says nothing to most people. The configs had roles in their names, stick with that please

Everything is now standardized to knb-bank, also "vpDefinitionId": "knb-bank",

sanderPostma committed:
DPP-89: PR feedback

Commit:
VDX-278 Fixed KVK broken pages:
   - Information Manual Request (readonly fields);
   - Credentials Issue Request (Layout);

sanderPostma committed:
DPP-89: Renamed PD

sanderPostma committed:
DPP-89: Reenabled OID4VP_WEBAPP_BASE_PATH

sanderPostma committed:
DPP-89: Reremoved OID4VP_WEBAPP_BASE_PATH

sanderPostma committed:
DPP-89: Added missing DBC PD

sanderPostma committed:
DPP-89: email -> emailAddress

sanderPostma committed:
DPP-89: Fixed button enable in info request page when all fields are already filled

sanderPostma committed:
DPP-89: Fixed button enable in info request page when all fields are already filled

sanderPostma committed:
DPP-89: Quick-fix deeplink

sanderPostma committed:
DPP-89: Fix mobile screen for issue credentials

sanderPostma committed:
DPP-89: Fix mobile screen for issue credentials

sanderPostma committed:
Merge remote-tracking branch 'origin/develop' into feature/DPP-89

# Conflicts:
#	packages/oid4vci-demo-frontend/src/components/Form/index.tsx
#	packages/oid4vci-demo-frontend/src/configs/kvk.json
#	packages/oid4vci-demo-frontend/src/configs/notary.json
#	packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx

sanderPostma committed:
DPP-89: reverted changes from DBC demo fixes that belong in the branch
